### ISSUE TITLE
Unable to put in configuration info for MS Azure TTS

### ISSUE THREAD

**Issue by NRS004** [karashiiro/TextToTalk#175]
Hello. Apologies if I missed something in the setup, but I think that I am encountering a bug when trying to setup Azure Cognitive Services. 

When opening the config window, I choose 'Azure Cognitive Services" which requests I put in a Region and Subscription key. When I type in either, as soon as that field loses focus it erases my input. I tried to still click save & login to see if it was just a visual bug, but it will not allow me to login. 

Hopefully this gif shows my issue:
[![FF14_Azure_TTS_Issue.gif](https://s11.gifyu.com/images/ScQxW.gif)](https://gifyu.com/image/ScQxW)

Happy to provide any additional info / log files as needed. Thanks!

**Comment by jagoss2142** [comment]
Having the same issue, opened an Azure account and tried signing in with the region and subscription key and just keeps deleting when trying to enter in

**Comment by karashiiro** [comment]
Should be fixed with https://github.com/goatcorp/DalamudPluginsD17/pull/2310

**Comment by jagoss2142** [comment]
> Should be fixed with [goatcorp/DalamudPluginsD17#2310](https://github.com/goatcorp/DalamudPluginsD17/pull/2310)

Tried deleting old version from plugins folder and plugin config folder and reinstalling new version and still having same issue. Also the Dalamud message in chat is saying what version TTT is loaded, it still says 1.24.2.1 even though the Dalamud plugins window is saying 1.27.2.0, unsure if its related to the main issue.


![Bug](https://github.com/karashiiro/TextToTalk/assets/53441315/17a8a99f-81fc-4bf0-a15b-5466e9e8c919)

![image](https://github.com/karashiiro/TextToTalk/assets/53441315/fbb7f263-178c-4e90-9962-08f8edd9f1ce)

**Comment by karashiiro** [comment]
Ah, for testing it should be fixed with https://github.com/goatcorp/DalamudPluginsD17/pull/2309 — is there an error when updating?

**Comment by jagoss2142** [comment]
Idk if I'm doing something wrong, I have the latest version of XIVLauncher v6.3.0 and it is Running Dalamud vD7.11.0.0 (I don't see a way to update this manually or even check for a new version of Dalamud). In-game I go to the Dalamud Plugin Installer and install TextToTalk:
![image](https://github.com/karashiiro/TextToTalk/assets/53441315/600275e3-5ffc-4b7b-9b25-b41fefe9b327)
The version says v1.24.2.1, so thinking that I need the latest just released version, I tried installing the testing build.

tl;dr
installing the plugin with the latest version of the launcher does not have the ACS fix, unsure if it just needs more time to take on the backend or what, but right now it is not working normally. 

Thank you for your hard work as well :)

**Comment by karashiiro** [comment]
Just checked and it has the correct versions right now, it may not have updated at that point, yeah
![image](https://github.com/karashiiro/TextToTalk/assets/49822414/17fcc0b8-a966-4a01-b1ed-cfec83c38d95)
